tools/pot/setup.py

```diff
@@ -16,17 +16,22 @@
 
 def generate_pot_version():
     try:
-        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
-        commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode()
-        return "custom_{}_{}".format(branch_name, commit_hash)
+        pot_dir = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(os.path.realpath(__file__))),
+                                                "openvino", "tools"))
+        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"], cwd=pot_dir)
```

Comment:
it seems duplication

Reply:
Yes, but we can't import anything in `setup.py` from `openvino.tools.pot `